### PR TITLE
[Merged by Bors] - feat: port Topology.Algebra.ContinuousAffineMap

### PR DESCRIPTION
**Pull request by int-y1**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by int-y1** [comment]
todo:

* [ ] 3 `norm_cast` errors
* [ ] `instance : Neg` failed to synthesize `ContinuousNeg W`
* [ ] 2 errors involving `toContinuousAffineMap`

**Comment by mattrobball** [comment]
There appears to be some global GitHub issues with pull requests. The commits making this green are pushed to origin but it appears things are not propagating here at the moment.

File: Mathlib/Topology/Algebra/ContinuousAffineMap.lean [hunk 1]
```diff
@@ -0,0 +1,292 @@
...
+@[norm_cast, simp]
+theorem coe_sub (f g : P →A[R] W) : ⇑(f - g) = f - g := rfl
+#align continuous_affine_map.coe_sub ContinuousAffineMap.coe_sub
+
+theorem sub_apply (f g : P →A[R] W) (x : P) : (f - g) x = f x - g x := rfl
+#align continuous_affine_map.sub_apply ContinuousAffineMap.sub_apply
+
+instance : Neg (P →A[R] W) :=
+  -- Porting note: Why does this instance need to be given explicitly
+  let _ : ContinuousNeg W := TopologicalAddGroup.toContinuousNeg
```
Comment (semorrison) [inline Mathlib/Topology/Algebra/ContinuousAffineMap.lean:241]:
```suggestion
```
It can be removed!

**Comment by semorrison** [comment]
bors d+

**Pull request closed without merging**